templates/schedule/create.html.twig

```diff
@@ -0,0 +1,59 @@
+{% extends "base.html.twig" %}
+
+{% block stylesheets %}
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
```

Comment:
I think bootstrap is supposed to be installed in another PR using webpack encore, I'll check tomorrow Europe/Paris